packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -197,7 +197,10 @@
         </div>
         <div class="sub-container">
             <div class="container-label">Table</div>
-            <NimbleTable @bind-Data="TableData" class="table"></NimbleTable>
+            <NimbleTable @bind-Data="TableData" class="table">
```

Comment:
I think we should keep our Angular & Blazor example apps fairly in line with each other. I'm not sure there's much value anymore in having all the different data types in the Angular example app when we just convert them all to strings. I'd be in favor of changing Angular to work like Blazor and only having first & last name columns.

Reply:
I've updated the example to just have a couple of string columns. I'll take a separate pass to make it work more like the Blazor app (use Faker for data).